src/ScriptTools.sol

```diff
@@ -32,9 +31,13 @@ library ScriptTools {
     string internal constant DEFAULT_DELIMITER = ",";
     string internal constant DELIMITER_OVERRIDE = "DSSTEST_ARRAY_DELIMITER";
 
-    function readInput(string memory input) internal view returns (string memory) {
+    function getRootChainId() internal returns (uint256) {
+        return vm.envOr("FOUNDRY_ROOT_CHAINID", uint256(1));
```

Comment:
So, this can't be `view`?

Reply:
`vm.envOr` is not a view. Actually this raises a good point. I thought this was what the `VmSafe` is for.

Reply:
Ahh nvm. I misread the explaination. `VmSafe` is only for operations that do not modify the VM's state. So this can still include write operations.

Reply:
Ok understood, not sure why `vm.envOr` is not `view`, but anyway it is what it is.